saison/parts/header.html

```diff
@@ -0,0 +1,11 @@
+<!-- wp:group {"style":{"spacing":{"padding":{"right":"6vw","left":"6vw","bottom":"6vh"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between","verticalAlignment":"center"}} -->
+<div class="wp-block-group" style="padding-right:6vw;padding-bottom:6vh;padding-left:6vw"><!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|20"}},"layout":{"type":"flex","orientation":"vertical"}} -->
+<div class="wp-block-group"><!-- wp:site-title /-->
+
+<!-- wp:site-tagline /--></div>
+<!-- /wp:group -->
+
+<!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+<div class="wp-block-group"><!-- wp:navigation {"ref":452,"overlayBackgroundColor":"base","overlayTextColor":"contrast"} /--></div>
+<!-- /wp:group --></div>
```

Comment:
Looks like this does hardcode a ref to a navigation menu.

@matiasbenedetto let me know if I should ping someone else, but since you've worked on CBT: is this meant to be handled by CBT? If not, should it be? Is there a workflow change Bea should make to address this issue? I can probably fix this manually for now, especially if there's a best practice. Thank you!